vmaas/webapp/vulnerabilities.py

```diff
@@ -49,7 +49,10 @@ def _evaluate_state(self, state, epoch, ver, rel, arch):
 
         candidate_arches = self.db_cache.ovalstate_id2arches.get(oval_state_id, [])
         if candidate_arches:
-            matched = matched and self.db_cache.arch2id[arch] in candidate_arches
+            if self.db_cache.arch2id.get(arch) is not None:
```

Comment:
I would say that more Python way for this condition is:
```if arch in self.db_cache.arch2id```

Reply:
Thanks for this suggestion. Did not know about it.